src/main/java/games/strategy/engine/ClientContext.java

```diff
@@ -8,13 +8,16 @@
 import games.strategy.triplea.settings.battle.options.BattleOptionsSettings;
 import games.strategy.triplea.settings.folders.FolderSettings;
 import games.strategy.triplea.settings.scrolling.ScrollSettings;
+import games.strategy.engine.framework.map.download.*;
```

Comment:
Probably should expand the list because it will be expanded anyway the next time someone runs Organize Imports/Clean Up.